osbs/schemas/container.json

```diff
@@ -241,8 +241,8 @@
           }
         },
         "modular_koji_tags": {
-          "description": "tags of modular Koji content to include in the build",
-          "type": ["array", "null"],
+          "description": "tags of modular Koji content to include in the build, if True is used, tag from target will be used",
```

Comment:
Thnks for changing this!

I'd be even more explicit here to say `tag from koji target will be used`!

Reply:
updated